Add Plan (#5)
olr-fh/ITOPS-Studienarbeit: chapter/Kundenfeedback.tex

%!TEX root = ../Thesis.tex
\section{Kundenfeedback (Amine)}

> Unser Unternehmen BlümchenHub ist ihm ganz wichtig, ihr Kunden zufrieden zu stellen. Dafür braucht unsere Softwareentwicklung zu verstehen was den Kunden gefällt und was nicht, deswegen ist es wichtig Feedback zu holen, um Bedürfnisse der Kunden berücksichtigen zu können.\newline

Reviewer: ```suggestion
Für unser Unternehmen BlümchenHub ist es wichtig, Kunden zufrieden zu stellen. Dafür muss unsere Softwareentwicklung verstehen was den Kunden gefällt und was nicht, deswegen ist es wichtig Feedback zu holen, um Bedürfnisse der Kunden berücksichtigen zu können.\newline
```

> Kundenfeedback sind Meinungen von Kunden nach ihrer Erfahrung mit einem Produkt, dies kann sowohl negativ als auch positiv sein. Das Sammeln von diesen Feedbacks hat eine große Priorität in BlümchenHub, da dies zu einer kontinuierlichen Verbesserung des Unternehmens und zur Analyse der Marktleistung führen kann.\footnote{Megha Adityan (2023). Warum Kundenfeedback wichtig ist. https://www.vtiger.com/de/blog/why-customer-feedback-is-important. Abgerufen am 08. Juni 2024.}

Reviewer: ```suggestion
Kundenfeedback sind Meinungen von Kunden nach ihrer Erfahrung mit einem Produkt, dies kann sowohl negativ als auch positiv sein. Das Sammeln von diesen Feedback hat eine große Priorität in BlümchenHub, da dies zu einer kontinuierlichen Verbesserung des Unternehmens und zur Analyse der Marktleistung führen kann.\footnote{Megha Adityan (2023). Warum Kundenfeedback wichtig ist. https://www.vtiger.com/de/blog/why-customer-feedback-is-important. Abgerufen am 08. Juni 2024.}
```

\subsection{Erfassung von Kundenfeedback}

> Feedback kann auch zwischen Mitarbeitern und Teams gesammelt werden. Dadurch können Probleme und Fehler während der Entwicklung schnell erkannt und behoben werden, um den Endbenutzern qualitativ hochwertige Software zu liefern. Um dies zu erreichen, können beispielsweise automatisierte Tests durchgeführt werden, indem während des gesamten Zyklus kontinuierlich Tests ausgeführt werden und gegebenenfalls Fehlermeldungen zurückgegeben werden, die dann behoben werden können. Darüber hinaus könnten Entwickler ihre Implementierungen untereinander austauschen und Code-Reviews erhalten, die Verbesserungsvorschläge machen können.

Reviewer: ```suggestion
Feedback kann auch von den Mitarbeitern und den Teams gesammelt werden. Dadurch können Probleme und Fehler während der Entwicklung schnell erkannt und behoben werden, um den Endbenutzern qualitativ hochwertige Software zu liefern. Um dies zu erreichen, können beispielsweise automatisierte Tests durchgeführt werden, indem während des gesamten Zyklus kontinuierlich Tests ausgeführt werden und gegebenenfalls Fehlermeldungen zurückgegeben werden, die dann behoben werden können. Darüber hinaus könnten Entwickler ihre Implementierungen untereinander austauschen und Code-Reviews erhalten, die Verbesserungsvorschläge machen können.
```

Dies kann jedoch nicht ausreichen, da das Produkt an die Bedürfnisse der Kunden angepasst werden muss. Die BlümchenHub2.0-Website enthält ein Feedback-Formular, eine einfache Umfrage, die am Ende des Besuchs erscheint. Dieses Formular enthält verschiedene Fragen, um die Zufriedenheit der Kunden mit dem Produkt zu beschreiben und auch Verbesserungsvorschläge zu machen.\footnote{Navya Cloudops (2023). DevOps Zero to Hero Day 13: Continuous Feedback and Monitoring. https://medium.com/@navya.cloudops/devops-zero-to-hero-day-13-continuous-feedback-monitoring-584bee738718. Abgerufen am 08. Juni 2024.}

\begin{figure}[hbt]
    \centering
    \includegraphics[width=0.7\textwidth]{img/JiraFeedbackFormular.png}
    \source{https://community.atlassian.com/t5/App-Central/5-Ways-to-Collect-Customer-Feedback-in-Jira/ba-p/2490973}
    \caption{Jira Issue Collector Feedback-Formular}
    \label{fig:JiraFeedbackFormular}
\end{figure}

> Damit Zeit und Aufwand gespart wird, wird Jira Issue Collector benutzt für die Sammlung der Kundenfeedbacks. Dies ist wie auf der \cref{fig:JiraFeedbackFormular} zu sehen in Form eines Buttons der gedrückt wird, damit das Formular angezeigt wird. So können alle Besucher der Website einfach ihr Feedback geben, ohne ein Jira-Konto zu haben. Sobald das Formular abgeschickt wird, erstellt Jira direkt ein Issue im Backlog.

Reviewer: ```suggestion
Damit Zeit und Aufwand gespart wird, wird der Jira Issue Collector benutzt, für die Sammlung der Kundenfeedbacks. Dies ist wie auf der \cref{fig:JiraFeedbackFormular} zu sehen in Form eines Buttons der gedrückt wird, damit das Formular angezeigt wird. So können alle Besucher der Website einfach ihr Feedback geben, ohne ein Jira-Konto zu haben. Sobald das Formular abgeschickt wird, erstellt Jira direkt ein Issue im Backlog.
```

> In Jira am Beispiel der \cref{fig:JiraIssueCollectorErstellen} wird zuerst ein Issue Collector erstellt, dies ist also Feedback gennant und enthält Fragen, die zu den Kunden gestellt werden. Danach wird eine Seite mit Codeschnipseln angezeigt., was wervendet wird, um den Issue Collector in der Website einzubinden.\footnote{https://support.atlassian.com/jira-cloud-administratio/docs/use-the-issue-collector. Abgerufen am 08. Juni 2024.}

Reviewer: ```suggestion
In Jira am Beispiel der \cref{fig:JiraIssueCollectorErstellen} wird zuerst ein Issue Collector erstellt, dies wird ein Feedback gennant und enthält Fragen, die den Kunden gestellt werden. Danach wird eine Seite mit Codeschnipseln angezeigt., was verwendet wird, um den Issue Collector in der Website einzubinden.\footnote{https://support.atlassian.com/jira-cloud-administratio/docs/use-the-issue-collector. Abgerufen am 08. Juni 2024.}
```
